dht.go

```diff
@@ -70,7 +70,7 @@ type IpfsDHT struct {
 	autoRefresh           bool
 	rtRefreshQueryTimeout time.Duration
 	rtRefreshPeriod       time.Duration
-	triggerRtRefresh      chan struct{}
+	triggerRtRefresh      chan chan<- error
```

Comment:
shouldn't this be just `chan error`?

Reply:
@vyzo The refresh event loop listens to the `triggerRtRefresh ` channel to decide when to refresh as seen [here](https://github.com/libp2p/go-libp2p-kad-dht/blob/6a005f4687485b7f3a8b18b18785d3b6b1e99b93/dht_bootstrap.go#L64).

So, we need to give it an error channel it can write the event loop error to.